### DISCUSSION TITLE
How do I roll back to older version (using Docker)?

### DISCUSSION THREAD

**Discussion by jepjoo** [open-webui/open-webui#1770] [Q&A]
I'm using Open-webui on Win11 via Docker Desktop. I installed 0.1.121 and seems like it broke my Oobabooga OpenAI compatible API setup. How do I roll back to 0.1.120?

(Open-webui connects and Oobabooga generates tokens when requested but no text appears on Open-webui screen. This was working well with previous version)

**Comment by justinh-rahb** [comment]
I can confirm that my LiteLLM-added configuration for Oobabooga isn't working either. Other LiteLLM models are.

```
ERROR:apps.litellm.main:500 Server Error: Internal Server Error for url: http://localhost:14365/v1/chat/completions
Traceback (most recent call last):
  File "/app/backend/apps/litellm/main.py", line 326, in proxy
    r.raise_for_status()
  File "/usr/local/lib/python3.11/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://localhost:14365/v1/chat/completions
INFO:     10.81.4.105:0 - "POST /litellm/api/v1/chat/completions HTTP/1.1" 500 Internal Server Error
ERROR:apps.litellm.main:500 Server Error: Internal Server Error for url: http://localhost:14365/v1/chat/completions
Traceback (most recent call last):
  File "/app/backend/apps/litellm/main.py", line 326, in proxy
    r.raise_for_status()
  File "/usr/local/lib/python3.11/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://localhost:14365/v1/chat/completions
INFO:     10.81.4.105:0 - "POST /litellm/api/v1/chat/completions HTTP/1.1" 500 Internal Server Error
```
![Screenshot 2024-04-26 at 8 58 00 AM](https://github.com/open-webui/open-webui/assets/52832301/808486ba-4d5f-4b45-bd0a-300720614db6)

The LiteLLM config snippet:
```yaml
model_list:
- litellm_params:
    api_base: http://host.docker.internal:5001/v1
    api_key: None
    model: gpt-3.5-turbo
  model_info:
    id: 82d07350-0f2a-4e42-85e9-f13bea8e4df5
  model_name: textgen.rahb.local
```

**Comment by jepjoo** [comment]
The issue was resolved in 0.1.122. OpenWebUI now displays text when using using OpenAI compatible API in Oobabooga.

Also, to answer my original question, to install an older version (for example v0.1.120) with Docker, "ghcr.io/open-webui/open-webui:main" needs to be changed to "ghcr.io/open-webui/open-webui:v0.1.120" in the installation command.

Full command would be:
`docker run -d --network=host -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:11434 --name open-webui --restart always ghcr.io/open-webui/open-webui:v0.1.120`
